decidim-core/app/helpers/decidim/translations_helper.rb

```diff
@@ -13,6 +13,7 @@ module TranslationsHelper
     def translated_attribute(attribute)
       attribute.try(:[], I18n.locale.to_s).presence ||
         attribute.try(:[], current_organization.default_locale).presence ||
+        attribute.try(:[], attribute.keys.first).presence ||
```

Comment:
I think specs are failing because `attribute` might be a plain `string` in some situations. This wasn't failing because the `[]` method doesn't exist on `attribute`.

We should probably be checking this before: `return attribute unless attribute.kind_of?(Hash)`

Reply:
And the method will look less ugly because we can remove all `try`s, right?